.github/workflows/deploy-auth.yml

```
@@ -0,0 +1,59 @@
+name: Deploy Auth Service
+
+on:
+  push:
+    branches: [ develop ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)